.github/workflows/ci-sql-cli.yaml

```diff
@@ -26,6 +26,22 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+# This list should only have non-sensitive env vars
+# Env vars with secrets should be in the specific jobs
+env:
+  GOOGLE_CLOUD_PROJECT: astronomer-dag-authoring
+  POSTGRES_HOST: postgres
+  POSTGRES_PORT: 5432
+  REDSHIFT_DATABASE: dev
+  REDSHIFT_HOST: utkarsh-cluster.cdru7mxqmtyx.us-east-2.redshift.amazonaws.com
```

Comment:
Doesn't it feel a bit dodgy we have a team members' name in the cluster name :joy:

Reply:
It does. But at least it is consistent 😅 We should change this for the python-sdk as well.

Reply:
Should we log a ticket for this?

Reply:
but what if our team member wants to have a cluster on their name :D ?